### PR TITLE
Add email on terminated option

### PR DESCRIPTION
**Pull request by keiran-rowell-unsw**

### CODE REVIEW

**Review by keiran-rowell-unsw — COMMENTED** [review]
whoops, should have @ad.unsw.edu to be safe, 'USER' will always resolve to zID on Katana

File: submit.yml.erb [hunk 1]
```diff
@@ -7,3 +7,6 @@ script:
     native:
         - "-l host=k095,ngpus=1,ncpus=8,mem=125gb,walltime=12:00:00"
     walltime: 43200
+    email:
+        - "<%= ENV['USER'] %>@unsw.edu.au"
```
Comment (nbtm-sh) [inline submit.yml.erb:11]: Needs to be `$USER@ad.unsw.edu.au`

**Comment by nbtm-sh** [comment]
@keiran-rowell-unsw you need to make the change as it will prevent me from approving this request if I push to this branch

**Review by nbtm-sh — CHANGES_REQUESTED** [review]

**Review by nbtm-sh — APPROVED** [review]
Looks good to me

**Pull request merged**